File: reverseSentence.js
function reverseSentence(sentence) {
  let sentenceArr = sentence.split(" ");
  let result = [];

  for (let i = sentenceArr.length - 1; i >= 0; i--) {
    result.push(sentenceArr[i]);
  }

  return result.join(" ");
}

console.log(reverseSentence("Man bites dog") == "dog bites Man");

File: reverseSentence.rb
def reverseSentence(sentence)
  sentenceArr = sentence.split(" ")
  result = []

  (sentenceArr.length - 1).downto(0).each do |i|
    result << sentenceArr[i]
  end

  return result.join(" ")
end

p reverseSentence("Man eats dog")
